<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCP_Client_A</class>
 <widget class="QMainWindow" name="TCP_Client_A">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCP_Client_A</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Listening</string>
    </property>
   </widget>
   <widget class="QLabel" name="receivedStatusLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="receivedProgressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="RetroArchLoc">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="PCSXLoc">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LoadSettings">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SaveSettings">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_NextChoice">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Next Choice</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
